controllers/event_handling_test.go

```diff
@@ -52,7 +53,7 @@ func TestEventHandler(t *testing.T) {
 		t.Fatalf("failed to create memory storage")
 	}
 
-	eventServer := server.NewEventServer("127.0.0.1:56789", logf.Log, k8sClient, true)
+	eventServer := server.NewEventServer("127.0.0.1:56789", logf.Log, k8sClient, true, true)
```

Comment:
Please expand the unit test to also cover when this is set to `false`.

Reply:
@gunishmatta apart from that it is LGTM from me.

Reply:
@pjbgf  I have a doubt, should I create a different event server on different port ?
Something like this 	
eventServerWithHttpDisabled := server.NewEventServer("127.0.0.1:56790", logf.Log, k8sClient, true, false)
or should I create a new test method?
Please advice on this.

Reply:
I'd recommend making this a table-driven test.